src/styles/blocks/stop-watch.scss

```diff
@@ -0,0 +1,50 @@
+.stop-watch {
+  width: 80vmin;
+  height: 80vmin;
+  justify-content: center;
+  align-items: center;
+  border: 10px dotted #aaa;
+  border-radius: 50%;
+  background-color: #000;
+
+  & .stop-watch__center {
+    position: relative;
+    width: 5vmin;
+    height: 5vmin;
+    z-index: 1;
+    background-color: #000;
+    border-radius: 50%;
+  }
+
+  & .stop-watch__minute-hand {
+    position: absolute;
+    width: 2vmin;
+    height: 30vmin;
+    bottom: 50%;
+    background-color: #aaa;
+    border-top-left-radius: 50%;
+    border-top-right-radius: 50%;
+
+    animation: spin 60s linear 0s calc(5 / 60) normal forwards;
+  }
+
+  & .stop-watch__second-hand {
+    position: absolute;
+    width: 1vmin;
+    height: 38vmin;
+    bottom: 50%;
+    background-color: #eee;
+    border-top-left-radius: 50%;
+    border-top-right-radius: 50%;
+
+    animation: spin 1s linear 0s 5 normal forwards;
```

Comment:
Change the animation properties to 'animation: spin 60s linear infinite;' for the correct rotation of the second hand.